docs/index.rst

```diff
@@ -1,6 +1,12 @@
 ``conbench``
 ============
 
+.. toctree::
+    :maxdepth: 1
+    :caption: The conbench philosophy:
```

Comment:
Thanks, this is a nice section.

Yep, I forgot to add documentation on how to iterate locally. I can add that in a followup PR.

Reply:
Addressed in #763